packages/common/react-components/src/components/Selector/Selector.tsx

```diff
@@ -5,9 +5,8 @@
 import { CaretDown, CaretUp } from 'phosphor-react';
 import React, { FC, KeyboardEvent, useState } from 'react';
 
-import { mx } from '@dxos/react-components';
-
-import { Input } from './Input';
+import { mx } from '../../util';
+import { Input } from '../Input';
```

Comment:
If this component does what I think it does, it needs to be reimplemented using Radix’s `Select` component which does a ton of the heavy lifting for us: https://www.radix-ui.com/docs/primitives/components/select

It can be styled using `tailwindcss-radix` which is part of our stack: https://tailwindcss-radix.vercel.app/#select